src/test/scala/com/lohika/vteraz/ValidatorTest.scala

```diff
@@ -0,0 +1,163 @@
+package com.lohika.vteraz
+
+import com.lohika.vteraz.validator.Person
+import com.lohika.vteraz.validator.Validator._
+import com.lohika.vteraz.validator.ValidatorConversion._
+import org.scalatest.FunSuite
+
+class ValidatorTest extends FunSuite {
+    test("isPositive Left if negative number specified") {
+        val result = positiveInt.validate(-2)
+        assert(!result.isRight)
```

Comment:
`assert(result.isLeft)` is same logically as `assert(!result.isRight)`, so one of these statements is redundant